src/navigators/DrawerNavigator.js

```diff
@@ -25,13 +26,21 @@ export type DrawerNavigatorConfig = {
 
 const { height, width } = Dimensions.get('window');
 
+const defaultContentComponent = (props: *) => (
+  <ScrollView alwaysBounceVertical={false}>
```

Comment:
should we set `flex: 1` on the scrollview? I believe we want it to take up the full drawer space available to it even if only one or two items

Reply:
Seems to set `flex: 1` by default, at least on iOS. In testing it looked fine.